py/surveyqa/summary.py

```diff
@@ -160,9 +161,31 @@ def last(arr):
         return arr[len(arr)-1]
     return exposures.group_by("TILEID").groups.aggregate(last)
 
+tzone = TimezoneInfo(utc_offset = -7*u.hour)
+t1 = Time(58821, format='mjd', scale='utc')
+t = t1.to_datetime(timezone=tzone)
+line_source = ColumnDataSource(data=dict(x=[t]))
+js = '''
+var geometry = cb_data['geometry'];
+console.log(geometry);
+var data = line_source.data;
+var x = data['x'];
+console.log(x);
+if (isFinite(geometry.x)) {
+  for (i = 0; i < x.length; i++) {
+    x[i] = geometry.x;
+  }
+  line_source.change.emit();
+}
+'''
```

Comment:
Please add a few comment to the above `line_source` and `js` code about what this does and why (target audience: a new collaborator reading this code 2 years from now and wanting to match up the code she is reading to the visual elements on the page).